resources/creatives/models/creatives.yml

```diff
@@ -14,10 +14,44 @@ postcardsCreative:
               $ref: "../../postcards/attributes/postcard_size.yml"
 
         front:
-          $ref: "../../postcards/attributes/front.yml"
```

Comment:
Are these files being referenced anywhere else? Maybe should be removed.

Reply:
They are being referenced within the postcard model.